examples/experimental/dagster-airlift/dagster_airlift/core/load_defs.py

```diff
@@ -119,10 +143,13 @@ def defs_with_assets_and_sensor(
     assets_defs: List[AssetsDefinition],
     airflow_instance: AirflowInstance,
     sensor_minimum_interval_seconds: int,
-    definitions_metadata: "_SerializedData",
+    definitions_metadata: "SerializableAirflowData",
+    airflow_data: "SerializableAirflowData",
```

Comment:
these names are unclear given what the type of arguments are. What is the data in each one? Why are there two sep ones?

Reply:
this was a mistake unfortunately. Not necessary for there to be two, thanks for catching.